_posts/fork/2020-8-18-Nord.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: theme
+title: Nord
+category: theme
```

Comment:
you don't need to submit two posts.

If your theme is forked from existing Typora theme, put it under `fork`
If not, put it under `theme`

Reply:
You can also test in our local.

https://theme.typora.io/doc/Submit-A-Theme/